src/autocomplete/autocomplete.ts

```diff
@@ -85,7 +89,10 @@ function getColumnSuggestions<
         realCursorTokenIndex,
         tokenDictionary,
     );
-    const result: Pick<AutocompleteResultBase, 'suggestColumns' | 'suggestColumnAliases'> = {};
+    const result: {
+        tableContextSuggestion?: TableContextSuggestion;
+        suggestColumnAliases?: AutocompleteResultBase['suggestColumnAliases'];
+    } = {};
```

Comment:
We need to create a type if we're using it in multiple places. Let's name it `SuggestionsContext` for example?